scripts/installscripts/install-jukebox.sh

```diff
@@ -982,9 +988,18 @@ install_main() {
         sudo chmod 440 "${sudoers_mopidy}"
     fi
 
+    # prepare lgpio build for bullseye as the binaries are broken
+    local pip_install_options=""
+    if [ "${OS_VERSION_ID}" -le "11" ]; then
+        ${apt_get} install swig unzip
+        mkdir -p tmp && cd tmp && wget -q http://abyz.me.uk/lg/lg.zip && unzip lg.zip > /dev/null && cd lg && make > /dev/null && sudo make install > /dev/null
```

Comment:
Do we need to pin a version here?

Reply:
You mean for lgpio? I didn't find a versioned zip file, only the one with latest and greatest, as described in the installation guide.